android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -85,11 +85,18 @@ class GestureHandlerOrchestrator(
   private fun hasOtherHandlerToWaitFor(handler: GestureHandler<*>) =
     gestureHandlers.any { !isFinished(it.state) && shouldHandlerWaitForOther(handler, it) }
 
-  private fun shouldBeCancelledByFinishedHandler(handler: GestureHandler<*>) = gestureHandlers.any { shouldHandlerWaitForOther(handler, it) && it.state == GestureHandler.STATE_END }
+  private fun shouldBeCancelledByFinishedHandler(handler: GestureHandler<*>) =
+    gestureHandlers.any { shouldHandlerWaitForOther(handler, it) && it.state == GestureHandler.STATE_END }
+
+  private fun handlersWithinHandlerBounds(handler: GestureHandler<*>) =
+    gestureHandlers.filter { it.isWithinBounds(wrapperView, handler.x, handler.y) }
```

Comment:
This doesn't seem to be doing what the name suggests
```suggestion
  private fun handlersWithinHandlerBounds(handler: GestureHandler<*>) =
    gestureHandlers.filter { handler.isWithinBounds(handler.view, it.x, it.y) }
```

- `isWithinBounds` checks whether the point is inside the passed view when handler's hitslop is also considered
- `wrapperView` is the root view, so I think this will return `true` in almost every case
- If the intention was `every handler that's tracking a pointer inside the view of the one passed as the argument` then I think the roles were swapped (though I haven't thought about that one for long 😅)

Reply:
i believe this one is resolved [here](https://github.com/software-mansion/react-native-gesture-handler/pull/3095#discussion_r1775280861)